src/com/rakovets/course/challenge/tasks_exception_handling/Task01.java

```diff
@@ -0,0 +1,14 @@
+package com.rakovets.course.challenge.tasks_exception_handling;
+
+public class Task01 {
+	public static void main(String[] args) {
+		try {
+			String vacuum = null;
+			vacuum.length();
+		} catch (Exception ex) {
```

Comment:
`Exception` is a broader concept than `NullPointerException`
Catch`NullPointerException`